setup.py

```diff
@@ -43,6 +44,7 @@
     entry_points={
         'console_scripts': [
             'cartography = cartography.cli:main',
+            'driftdetect = driftdetect.cli.main'
```

Comment:
This needs to be updated to reflect the package change. I'd also suggest `cartography-drift` or `cartography-detectdrift` for the console script name so that it shares a prefix with the regular cartography console script.